specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/patchComputeQuotaRequest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus",
+    "resourceName": "standardFSv2Family",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "unit": "Count",
+        "name": {
+          "value": "standardFSv2Family"
+        }
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "location": "https://management.azure.com/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605?api-version=2021-03-15-preview",
+        "Retry-After": "30",
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/operationsStatus/2B5C8515-37D8-4B6A-879B-CD641A2CF605?api-version=2021-03-15-preview"
+      },
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
```

Comment:
>/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605", [](http://example.com/codeflow?start=15&length=181)

This is the resource ID not ID for async operation. The client should get URL for tracking async operation from headers not from this ID.

It should be '/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/Microsoft.Quota/quotaLimits/standardFSv2Family'

Reply:
If Microsoft.Quota/quotaLimitsRequests resource ID needs to be returned, please add a property like 'quotaLimitsRequestId'.

Reply:
I will update all the examples.

Reply:
Also I'm not sure if returning the request ID here. What happens if the resource is updated (PATCH) later? Does the request ID change?

Reply:
Updated the examples to resourceId.